hack/get_ci_vm.sh

```diff
@@ -158,7 +175,7 @@ fi
 
 # Couldn't make rsync work with gcloud's ssh wrapper :(
 TARBALL_BASENAME=$VMNAME.tar.bz2
-TARBALL=/tmp/$TARBALL_BASENAME
+TARBALL=$HOME/$TARBALL_BASENAME  # inside container $HOME == $TMPDIR
```

Comment:
This doesn't work: scp fails because there's no `$HOME` on the destination (`root@$VMNAME`). An easy fix (testing it now) is, on line 216:
```
- showrun $SCP_CMD $TARBALL                root@$VMNAME:$TARBALL
+ showrun $SCP_CMD $HOME/$TARBALL_BASENAME root@$VMNAME:$TARBALL
```

Reply:
Ahh, this must be what @baude hit also: https://paste.fedoraproject.org/paste/JNPN3oZmsoK9H1WnXeHs9A

Reply:
ohhh ya, u right, I see the problem.   Ya I think we can just drop $TARBALL and only use $TARBALL_BASENAME along with the correct path.

Reply:
I was just writing that same comment: that's my inclination also. But please just rename it to `$TARBALL`; keeping BASENAME is no longer useful.

Reply:
okay, this seems to fix the problem.  squashing and pushing.